gatsby-config.js

```diff
@@ -587,70 +587,6 @@ module.exports = {
                     }
                 ]
             },
-            {
-                title: "Adobe Experience Platform Assurance",
-                path: "/documentation/platform-assurance",
```

Comment:
Is there a way to set up redirects for these paths for a while until the users bookmark the new links?

Reply:
I think that would be good, but I'm unsure how redirects work 😅